src/disable-amp.js

```diff
@@ -12,9 +12,9 @@ function observeDomChanges(callback) {
  * @param amp element
  */
 const hideAmpIcon = (amp) => {
-    const ampIcon = amp.querySelector('[aria-label="AMP logo"]');
+    const ampIcon = amp.querySelector('[aria-label="Logo AMP"]');
```

Comment:
do you have it as `Logo AMP`?
![image](https://user-images.githubusercontent.com/15062885/82256650-04cd7680-9957-11ea-89c1-a047cc90cd89.png)